pkg/project/project.go

```diff
@@ -114,8 +113,36 @@ func (p *Project) CommonSetupSteps() error {
 	return nil
 }
 
+func (p *Project) CommonTearDownSteps() error {
+	p.logger.Log("info", "starting common teardown steps")
+	steps := []Step{
+		Step{
+			Run: "helm delete --purge sonobuoy-chart || true",
+		},
+		Step{
+			Run: "helm reset --force",
+		},
+		Step{
+			Run: "kubectl delete clusterrolebinding permissive-binding",
+		},
+		Step{
+			Run: "kubectl -n kube-system delete sa tiller",
+		},
+		Step{
+			Run: "kubectl delete clusterrolebinding tiller",
```

Comment:
I think it'd be nice to use `--context=minikube` explicitly. I'm pretty sure many ppl will forget to set default context to minikube before running those tests.

Reply:
That wouldn't work with the helm commands, instead I've added a `kubectl config use-context minikube` as the first step in the common setup tasks, pls take a look